website/docs/r/sql_failover_group.html.markdown

```diff
@@ -101,15 +101,17 @@ The following arguments are supported:
 
 The following attributes are exported:
 
-* `id` - The failover group ID
-* `location` - the location of a SQL server in `partner_servers`
-* `role` - the current role of a SQL server in `partner_servers`
-* `role` - the current role of the SQL server named in `server_name`
+* `id` - The failover group ID.
+* `location` - the location of the failover group.
+* `server_name` - the name of the primary SQL Database Server.
+* `role` - local replication role of the failover group instance.
+* `databases` - list of databases in the failover group.
+* `partmer_servers` - list of partner server information for the failover group.
```

Comment:
I think this should be?

```suggestion
* `partner_servers` - list of partner server information for the failover group.
```

Reply:
Thanks for the feedback. The typo has been fixed.